integration/SimpleCacheClient.test.ts

```diff
@@ -4,6 +4,7 @@ import * as os from 'os';
 import {SimpleCacheClient, TimeoutError} from '../src';
 import {AlreadyExistsError, NotFoundError} from '../src/Errors';
 import {TextEncoder} from 'util';
+import { ListSigningKeysResponse } from '../src/messages/ListSigningKeysResponse';
```

Comment:
nit: remove spaces around {} to be consistent in formatting?

Reply:
Oh good catch, I'm surprised `eslint` didn't fix this. Thanks, I'll fix it!